### PR TITLE
Bump helm.sh/helm/v3 from 3.3.4 to 3.4.2 (By updating helmclient) 

### PR DESCRIPTION
**Pull request by tomahawk28**
## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/giantswarm/appcatalog v0.3.2
 	github.com/giantswarm/backoff v0.2.0
 	github.com/giantswarm/exporterkit v0.2.0
-	github.com/giantswarm/helmclient/v4 v4.0.0
+	github.com/giantswarm/helmclient/v4 v4.0.1-0.20201214091549-6e0fd3ddb220
```
Comment (tomahawk28) [inline go.mod:11]: TODO: change it after release new helmclient.

File: go.mod [hunk 2]
```diff
@@ -34,8 +34,15 @@ replace (
 	github.com/bketelsen/crypt => github.com/bketelsen/crypt v0.0.3
 	// Use moby v20.10.0-beta1 to fix build issue on darwin.
 	github.com/docker/docker => github.com/moby/moby v20.10.0-beta1+incompatible
+
+	// Use go-logr/logr v0.1.0 since some they have breaking changes other component couldn't apply
```
Comment (rossf7) [inline go.mod:38]:
```suggestion
	// Use go-logr/logr v0.1.0 due to breaking changes in v0.2.0 that can't be applied.
```

**Review by rossf7 — APPROVED** [review]
LGTM

File: go.mod (hunk 2 above)
Comment (tomahawk28) [reply go.mod:38]: This is better, Thanks :)

**Pull request merged**